### DISCUSSION TITLE
Field 'browser' doesn't contain a valid alias configuration

### DISCUSSION THREAD

**Discussion by budarin** [pmndrs/zustand#2223] [Bug report] [answered]
We keep all dependencies for our projects in our installable package. It contains all the packages necessary for typical projects, including zustand.

After upgrading some dependencies
from
```
"@babel/core": "^7.23.2",
"@babel/eslint-plugin": "^7.22.10",
"@babel/preset-env": "^7.23.2",
"zustand": "^4.4.5",
```
to
```
"@babel/core": "^7.23.5",
"@babel/eslint-plugin": "^7.23.5",
"@babel/preset-env": "^7.23.5",
"zustand": "^4.4.7",
```
We have the error while building the project with new deps
![image](https://github.com/pmndrs/zustand/assets/8055157/58ed36b1-6673-4b83-a93e-07d11a9f79b9)

adding zustand directly to the project solves this problem momentarily, but this does not suit us - we need all the dependencies necessary for react projects to arrive in our package.

I suspect that the problem is probably related to #2170 in [v4.4.6]

**Comment by dai-shi** [comment]
That's interesting. I'm not sure what we can do.

Just to confirm, can you revert `@babel/*` and only change `zustand` to `4.4.6`?

**Reply by budarin** [reply]
I confirm - an error in v4.4.6
rolled back babel and installed v4.4.6 - the error is present

**Reply by dbritto-dev** [reply]
@budarin would you mind sharing us a minimal repro? I would like to help

**Reply by budarin** [reply]
@dbritto-dev 

It's pretty hard to find the time to create a project for demonstration, but I'll try

And in short: there is a package @budarin/spa-react-app-config - simply add it as dependency and try to build the project.

At the same time, you need to make sure that zustand is not installed in the project itself, but will be added to node_modules by this package

**Comment by budarin** [comment, marked as answer]
solved!

the problem lay in node_modules: for some reason, zustand was not installed when installing the package in which it was specified in dependencies

After removing node_modules, yarn.lock and reinstalling all dependencies - zustand appeared in node_modules
